src/base/bittorrent/torrenthandle.h

```diff
@@ -214,6 +211,7 @@ namespace BitTorrent
         virtual bool hasFilteredPieces() const = 0;
         virtual int queuePosition() const = 0;
         virtual QVector<TrackerEntry> trackers() const = 0;
+        virtual bool tracker(const QString &trackerUrl, TrackerEntry &entry) const = 0;
```

Comment:
Please rename it to `findTrackerEntry` or `trackerEntryByURL`.

Reply:
Also I would modify TrackerEntry class to have "valid" property (i.e. default constructed instance is invalid). Then this method can have simpler signature, i.e. accept URL and return TrackerEntry.

Reply:
After my recent changes it is again used only in one place, I remove it.